<?xml version="1.0"?>
<launch>
    <machine name="beaglebone" address="beaglebone" env-loader="/opt/ros/indigo/envBBB.sh" user="ubuntu" password="KR5sixx" />
    <!-- Motor Q5 (1)-->
    <arg name="Name_M1" value="Q5" />
    <param name="$(arg Name_M1)_dev" value="ttyO4" />
    <!--param name="$(arg Name_M1)_init_pose" value="0" /-->
    <param name="$(arg Name_M1)_enc_limit" value="99413" />
    <param name="$(arg Name_M1)_angle_limit" value="85" />
    <param name="$(arg Name_M1)_transmission" value="205.588" />

    <arg name="Name_M2" value="Q4" />
    <param name="$(arg Name_M2)_dev" value="ttyO5" />
    <param name="$(arg Name_M2)_enc_limit" value="120000" />
    <param name="$(arg Name_M2)_angle_limit" value="90" />
    <param name="$(arg Name_M2)_transmission" value="196" />

    <arg name="Name_M3" value="Q6N" />
    <param name="$(arg Name_M3)_dev" value="ttyO1" />
    <!--param name="$(arg Name_M3)_init_pose" value="0" /-->
    <param name="$(arg Name_M3)_enc_limit" value="101354" />
    <param name="$(arg Name_M3)_angle_limit" value="90" />
    <param name="$(arg Name_M3)_transmission" value="235.801" />

    <arg name="Name_M4" value="Q6P" />
    <param name="$(arg Name_M4)_dev" value="ttyO2" />
    <!--param name="$(arg Name_M4)_init_pose" value="0" /-->
    <param name="$(arg Name_M4)_enc_limit" value="114023" />
    <param name="$(arg Name_M4)_angle_limit" value="90" />
    <param name="$(arg Name_M4)_transmission" value="209.601" />

    <arg name="BaudRate" value="115200" />

    <node machine="beaglebone" pkg="faulhaber_driver" type="server" name="faulhaber_server" args="$(arg Name_M1) $(arg Name_M2) $(arg Name_M3) $(arg Name_M4) $(arg BaudRate)" output="screen" />
</launch>

